src/njoy21/input/POWR/Lib3/src/readCard9.hpp

```diff
@@ -0,0 +1,13 @@
+template< typename Istream >
+optional< std::vector< Card9 > > readCard9( Istream& is, const int if5,
+                                            const size_t ntis ){
+  if( if5 != 2 ) return std::nullopt;
+
+  std::vector< Card9 > card9List;
```

Comment:
This should be the first line of the function to enable the compiler to perform optimizations.

Reply:
Ask Austin (if you can find him) about the specifics.

Reply:
I assume since the first line is a branching point that may eliminate the need for that variable, it is better to put it after the possibility that it won't be used is determined.  A brief google search was uninformative in this respect.  I'll shoot an email to Austin's black hole of an inbox and see what his thoughts are.